style.css

```diff
@@ -0,0 +1,86 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap");
+
+body {
+  font-family: "Poppins", sans-serif;
+}
+
+header {
+  display: flex;
+  margin: 5px;
+  padding: 5px;
+}
```

Comment:
- [ ] As mentioned by the previous reviewer their is a background image in the figma design, I would suggest you also put it your background. You can always export icons and background from figma. Use [this](https://help.figma.com/hc/en-us/articles/360040028114-Guide-to-exports-in-Figma) document to see how to export backgrounds and images from Figma

**Arrows point to the background**
<img width="379" alt="Screenshot 2023-02-15 at 4 34 33 AM" src="https://user-images.githubusercontent.com/44798044/218888532-96626e16-48ee-4b6d-b36c-16c7c4793950.png">

Reply:
- [ ] Please follow the design and also implement the changes requested by @shella12 .
Your current design doesn't yet match the Figma template and you have not implemented the requested change. 

**Your current design**
![nobg](https://user-images.githubusercontent.com/99656687/218991713-977787ea-c49f-4c2f-9514-6d641ebac91d.png)

- [ ] Please add the background image in your CSS file. You can use [this resource](https://www.w3schools.com/cssref/pr_background-image.php)  to learn how to add background images